packages/connectivity/src/scp-cf/token-accessor.spec.ts

```diff
@@ -267,6 +267,19 @@ describe('token accessor', () => {
       expect(retrieveFromCacheSpy).toHaveBeenCalledTimes(0);
     });
 
+    it('throws an error having the cause, not the config property', async () => {
+      mockClientCredentialsGrantCall(
+        providerXsuaaUrl,
+        { access_token: signedJwt({ dummy: 'content' }) },
+        401,
+        destinationBindingClientSecretMock.credentials
+      );
+      await serviceToken('destination').catch(error => {
```

Comment:
`.catch(expect(...))` will give a false positive if there is no error thrown at all since `.catch` is never executed. For example, if `200` is returned instead of `401`, the test will still pass.

Try to use `rejects` instead:
```suggestion
      const promise = serviceToken('destination');
      await expect(promise).rejects.toHaveProperty('cause');
      await expect(promise).rejects.not.toHaveProperty('config');
```

Reply:
thanks. It must be tested with `.rejects`. I removed the try-catch block.